sos/plugins/filesys.py

```diff
@@ -38,7 +38,8 @@ def setup(self):
         self.add_cmd_output([
             "df -ali -x autofs",
             "findmnt",
-            "lslocks"
+            "lslocks",
+            "ls -ltradZ /tmp"
```

Comment:
What about systems where SELinux is disabled at *build* time? Presumably we'll get a file with an error in it - not the end of the world, but it's always good to avoid leaving those lying around.

Part of me is inclined to say that we should move this into a new `RedHatFilesys` class: we know that all Red Hat builds all enable the `-Z` switch but other distributions may not.